zarr/tests/test_storage.py

```diff
@@ -259,9 +307,12 @@ def test_getsize(self):
 
         store.close()
 
-
 class StoreTests(MutableMappingStoreTests):
     """Abstract store tests."""
+    
```

Comment:
IIRC, I needed to move this one, because LRUChunkCache doesn't implement the contextmanager interface - maybe implement it instead?